src/devices/Rangefinder2DWrapper/CMakeLists.txt

```diff
@@ -36,3 +36,70 @@ if(NOT SKIP_Rangefinder2DWrapper)
 
   set_property(TARGET yarp_Rangefinder2DWrapper PROPERTY FOLDER "Plugins/Device")
 endif()
+
+####################################################################################################
+yarp_prepare_plugin(Rangefinder2DWrapperROS
+                    CATEGORY device
+                    TYPE Rangefinder2DWrapperROS
+                    INCLUDE Rangefinder2DWrapperROS.h
```

Comment:
Since this is a wrapper, I think that this should have `WRAPPER Rangefinder2DWrapperROS`
Or maybe, we could have a look at how this works and see if we can enhance this mechanism for ros support?

Reply:
done